zilliqa/src/cfg.rs

```diff
@@ -159,6 +159,10 @@ pub struct NodeConfig {
     /// Maximum allowed RPC response size
     #[serde(default = "max_rpc_response_size_default")]
     pub max_rpc_response_size: u32,
+    /// The block height at which ZQ1 converted persistence ends. After this block ZQ2 blocks begin.
+    /// This value should be required only for proto networks to distinguise between ZQ1 and ZQ2 blocks. In future converted networks all ZQ1 blocks will be distinguishable by their zeroed state root hash.  
```

Comment:
> In future converted networks all ZQ1 blocks will be distinguishable by their zeroed state root hash

I would caution against this comment for two reasons:

1. When pruning is implemented ZQ2 blocks might have some kind of 'zero' state hash to indicate we haven't stored their state.
2. Eventually ZQ1 blocks won't have a zero state root hash.

I don't want people to rely upon the property that `block.is_from_zq1() <=> block.state_root_hash.is_zero()`

Reply:
Okay understood thanks